Documents/How-to-use.md

```diff
@@ -207,48 +207,61 @@ Per Category, each XML file has VM name, Resource Group name, etc. We do not rec
             CentOS Linux release 7.3.1611 (Core)
             CentOS Linux release 7.4.1708 (Core)
             CentOS Linux release 7.5.1804 (Core)
+            CentOS Linux release 7.6
+            CentOS Linux release 7.7
+            CentOS Linux release 8.0
+            CentOS Linux release 8.1
             CentOS release 6.5 (Final)
             CentOS release 6.6 (Final)
             CentOS release 6.7 (Final)
             CentOS release 6.8 (Final)
             CentOS release 6.9 (Final)
             CentOS release 6.10 (Final)
         b. ClearLinux
-            Clear Linux OS for Intel Architecture 1
+            Clear Linux OS for Basic
+            Clear Linux OS for Containers
+            Clear Linux OS for Machine-Learning
         c. Debian
-            Debian GNU/Linux 7 (wheezy)
-            Debian GNU/Linux 8 (jessie)
-            Debian GNU/Linux 9 (stretch)
-        d. OpenSUSE
-            openSUSE Leap 42.3
-        e. Oracle
+            Debian GNU/Linux 10 (+backport)
+            Debian GNU/Linux 11
```

Comment:
Don't we support for Debian 7,8,9? Do we need to fix in master branch?

Reply:
I don't think we support those version.